external/src/main/java/edu/illinois/cs/cogcomp/pipeline/common/ExternalToolsConfigurator.java

```diff
@@ -32,7 +32,7 @@
      * individually, then splice the annotations together. This allows for partial annotation of
      * documents in cases where document text causes local problems for individual annotators.
      */
-    public static final Property USE_SENTENCE_PIPELINE = new Property("useSentencePipeline", FALSE);
+    public static final Property USE_SENTENCE_PIPELINE = new Property("useSentencePipeline", TRUE);
```

Comment:
Ah shoot. You're right; the order in the factory file was reverse. Fixed it now.